package.json

```diff
@@ -71,17 +71,17 @@
     "husky": "^8.0.2",
     "lint-staged": "^13.0.3",
     "lodash": "^4.17.21",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^3.3.0",
```

Comment:
```suggestion
```

this was used to generate the data feeds in our CI, i don't think this is necessary anymore

Reply:
Removed